src/upgrade.c

```diff
@@ -158,6 +164,16 @@ static int upgrade_main(void *ctx, struct apk_database *db, struct apk_string_ar
 	if (uctx->self_upgrade_only)
 		return 0;
 
+  if (uctx->ignore) {
+    char **pkg_name;
+    struct apk_name *name;
+    foreach_array_item(pkg_name, args) {
+      name = apk_db_get_name(db, APK_BLOB_STR(*pkg_name));
+      apk_message("Ignoring upgrade : %s",name->name);
```

Comment:
I don't think we want this verbose messages here. This could be debug message or just be removed.

Reply:
IMO it's worth it to keep this line as debug message but I can move it further in the process if required (somewhere in the solver for example)

Reply:
Finally, I just removed this message since I didn't find an accurate place to put the debug message.